templates/default/etc/init.d/node-ff.erb

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh
+
+#
+# chkconfig: - 80 20
+# processname: node-ff
+# description: Selenium Grid node for Firefox
+#
+
+### BEGIN INIT INFO
+# Provides: Selenium Grid Firefox node
+# Required-Start: $network $syslog
+# Required-Stop: $network $syslog
+# Default-Start:
+# Default-Stop:
+### END INIT INFO
+
+PATH_TO_JAR=<%= node['aet']['seleniumgrid']['node_ff']['root_dir'] %>/selenium-server-standalone-3.8.1.jar
+HUB_URL="http://localhost:4444"
+CAPABILITIES="browserName=firefox,version=ANY,platform=LINUX,maxInstances=5"
+LOG_DIR_PATH=<%= node['aet']['seleniumgrid']['node_ff']['log_dir'] %>
+
+hub_pid()
+{
+  echo `ps -aefw | grep java | grep -e '-role node' | awk '{print $2}'`
+}
+
+start() {
+  pid=$(hub_pid)
+  if [ -n "$pid" ]
+  then
+    echo "Selenium Grid Firefox node is already running (pid: $pid)"
+  else
+    echo "Starting Selenium Grid Firefox node"
+    nohup java -jar $PATH_TO_JAR -role node -hub $HUB_URL -browser $CAPABILITIES >> $LOG_DIR_PATH/node-ff.log 2>&1 &
```

Comment:
I like this part here:  `2>&1` :) Could we have the same in the hub service template ?

See https://en.wikipedia.org/wiki/Redirection_(computing)#Redirecting_to_and_from_the_standard_file_handles